multiqc/modules/hicpro/hicpro.py

```diff
@@ -289,15 +306,24 @@ def hicpro_mapping_chart(self):
         data = [{}, {}]
         for s_name in self.hicpro_data:
             for r in [1, 2]:
-                try:
-                    data[r - 1][f"{s_name} [R{r}]"] = {
-                        "Full_Alignments_Read": self.hicpro_data[s_name][f"global_R{r}"],
-                        "Trimmed_Alignments_Read": self.hicpro_data[s_name][f"local_R{r}"],
-                        "Failed_To_Align_Read": int(self.hicpro_data[s_name][f"total_R{r}"])
-                        - int(self.hicpro_data[s_name][f"mapped_R{r}"]),
-                    }
-                except KeyError as e:
-                    log.error(f"Missing expected plot key {e} in {s_name} Read {r}")
+                n_global = self.hicpro_data[s_name].get(f"global_R{r}")
+                n_local = self.hicpro_data[s_name].get(f"local_R{r}")
+                n_mapped = self.hicpro_data[s_name].get(f"mapped_R{r}")
+                n_total = self.hicpro_data[s_name].get(f"total_R{r}")
+                d = dict()
+                if n_global is not None:
+                    d["Full_Alignments_Read"] = n_global
+                if n_local is not None:
+                    d["Trimmed_Alignments_Read"] = n_local
+                if n_mapped is not None and n_total is not None:
+                    try:
+                        d["Failed_To_Align_Read"] = int(n_total) - int(n_mapped)
+                    except ValueError:
+                        log.debug(
```

Comment:
This shouldn't happen very often, right?
```suggestion
                        log.warning(
```

Reply:
Yeah, agree